docs/product-release-notes/src/main/asciidoc/rn-rhpam-701-con.adoc

```diff
@@ -1,17 +1,17 @@
 [id='rn-rhpam-701-con']
-= Release notes for {PRODUCT} 7.0.1
+= Fixed issues in {PRODUCT} 7.0.1
 
 {PRODUCT} 7.0.1 provides increased stability and fixed issues listed in this section.
 
 == {CENTRAL}
 
-* Updating task details alters Due On time [https://issues.jboss.org/browse/RHPAM-1229[RHPAM-1229]]
+//* Updating task details alters Due On time [https://issues.jboss.org/browse/RHPAM-1229[RHPAM-1229]]
 * Support up to 12 arguments in RHS of executable model [https://issues.jboss.org/browse/RHPAM-1190[RHPAM-1190]]
 * Duplicated or imported project should use the suffix "-2" instead of " [2]" [https://issues.jboss.org/browse/RHPAM-1176[RHPAM-1176]]
 * Library content is not visible when Projects link is clicked [https://issues.jboss.org/browse/RHPAM-1110[RHPAM-1110]]
 * Manage list views to do not apply default filters [https://issues.jboss.org/browse/RHPAM-1109[RHPAM-1109]]
 * Process Designer task name is for internal use and must not be editable for ServiceTasks [https://issues.jboss.org/browse/RHPAM-1104[RHPAM-1104]]
-* Space screen is shown sometimes after an asset is closed [https://issues.jboss.org/browse/RHPAM-1091[RHPAM-1091]]
+* Space screen is sometimes show after an asset is closed [https://issues.jboss.org/browse/RHPAM-1091[RHPAM-1091]]
```

Comment:
This seems to have introduced a typo - but take my linguistic comments with a grain of salt ;)

Reply:
Nope you right. Thanks for the catch!